header.php

```diff
@@ -25,17 +42,32 @@
                         <li class="menu-item">
                             <a href="<?php echo SITE_URL ?>index.php">Home</a>                 
                         </li>
-<?php // Here i have to check if the user is logged.
-// If yes we need the see Logout and update acc ?>
-                        <li class="menu-item">
-                            <a href="<?php echo SITE_URL ?>register.php">Register</a>
-                        </li>
                         <li class="menu-item">
-                            <a href="<?php echo SITE_URL ?>login.php">Login</a>                    
-                        </li>
-
-
-                    </ul>
+                        <?php
+                        if (!$_SESSION['logged_in']) {
+                            ?>
+                    <a href="<?php echo SITE_URL ?>register.php">Register</a>
+                        <?php } else {
+                            if ($_SESSION['is_company'] == true) {
+                                ?>
+                    <a href="<?php echo SITE_URL ?>dashboard.php">Dashboard</a>
+                    <a href="<?php echo SITE_URL ?>actions-job.php">Create Job</a>
+                            <?php }
+                            ?>
+                    <a href="<?php echo SITE_URL ?>profile.php">My Profile</a>
+                        <?php }
+                        ?>
+            </li>
```

Comment:
The formatting here is horrendous, could you fix it?

When dealing with conditions inside your HTML, you can use the following formatting:
```suggestion
                        <?php
                        if ( ! isset ( $_SESSION['logged_in'] ) ) :
                            ?>
                            <a href="<?php echo SITE_URL ?>register.php">Register</a>
                            <?php
                        else :
                            if ($_SESSION['is_company'] == true) :
                                ?>
                                <a href="<?php echo SITE_URL ?>dashboard.php">Dashboard</a>
                                <a href="<?php echo SITE_URL ?>actions-job.php">Create Job</a>
                                <?php
                            endif;
                            ?>
                            <a href="<?php echo SITE_URL ?>profile.php">My Profile</a>
                            <?php
                        endif;
                        ?>
            </li>
```

Reply:
Formatting is really important for readability and when you start with WordPress you will be required to follow a set of formatting rules. So it's good to start earlier with that.